lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -329,7 +332,7 @@ def set_aspect(self, aspect, adjustable=None, anchor=None, share=False):
         super().set_aspect(
             aspect, adjustable=adjustable, anchor=anchor, share=share)
 
-    def set_box_aspect(self, aspect, *, zoom=1):
+    def set_box_aspect(self, aspect=None, *, zoom=None):
```

Comment:
`aspect=None`: There has already been a pseudo-default `None`, but users would have to explicitly write `set_box_aspect(None)`. While I agree that's a bit awkward, IMHO `set_box_aspect()` is not better. For a `set_*` method that's not a bool, I'd always expect an explict value to be set.

`zoom=None`: This is an API change. I'm not very familiar with the 3d code. Is it reasonable to decouple aspect and zoom changes? Before `set_box_aspect()` always sets *aspect* and *zoom*, which is easy to understand. Note that the None parameters for aspect and zoom now would have a different semantics "use fixed default 4:4:3" vs. "don't modify the existing value", which can be rather confusing. If we want such a decoupling, maybe splitting in `set_box_aspect(aspect)` and `set_zoom(zoom)` would be better.


`set_box_aspect()` defines both, but when leaving out `zoom`, you'll keep whatever you had before.

Reply:
Points taken, I'll revert the API changes. The API here is rather clunky IMO, but I'd want to think more about a comprehensive rewrite of the interface before putting piecemeal changes in.